CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [ENHANCEMENT] Assert ingestion rate limits as early as possible [#2640](https://github.com/grafana/tempo/pull/2703) (@mghildiy)
 * [ENHANCEMENT] Add several metrics-generator fields to user-configurable overrides [#2711](https://github.com/grafana/tempo/pull/2711) (@kvrhdn)
 * [BUGFIX] Fix panic in metrics summary api [#2738](https://github.com/grafana/tempo/pull/2738) (@mdisibio)
+* [FEATURE] Add filtering support to Generic Forwarding [#2742](https://github.com/grafana/tempo/pull/2742) (@Blinkuu)
```

Comment:
Nit: we group entries by type, so this one should be underneath the other `[FEATURE]`'s right above `[ENHANCEMENT]`

Reply:
Updated in [25f4ef4](https://github.com/grafana/tempo/pull/2742/commits/25f4ef4390e7c39e2de249555008ae381476d678).